### PR TITLE
Add Rails 6.1 to the testing matrix

### PR DESCRIPTION
**Pull request by derekprior**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,11 +13,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["2.7","2.4"]
-        rails: ["5.2", "6.0", "master"]
+        ruby: ["2.7", "2.4"]
```
Comment (calebhearth) [inline .github/workflows/ci.yml:16]: I propose
```suggestion
        ruby: ["ruby-head", "3.0", "2.7"]
```

Comment (calebhearth) [reply .github/workflows/ci.yml:16]: or master or whatever it is for ruby

**Comment by calebhearth** [comment]
You'll need to skip Rails 6+ on Ruby 2.7

**Comment by derekprior** [comment]
Having bigger problems at the moment -- the tests are broken on 6.1. I think I have a guess as to why but I suspect my fix might break older versions...

**Pull request merged**